Eat_and_Greet/src/screens/LoginScreen/index.tsx

```diff
@@ -14,8 +14,8 @@ function LoginScreen() {
       <View style={styles.ContentContainer}>
         <StatusBar style="auto" />
         <Image style={LoginStyles.logoStyles} source={require('../../../assets/images/logo.png')}/>
-        <RoundedHomeButton name="Log In" top="5%" onPress={() => navigation.navigate('Home')}/>
```

Comment:
Added the margins to RoundedHomeButton which removed the need for these positioning parts on the login screen.